src/pool/srv_pool.c

```diff
@@ -5413,15 +5422,15 @@ pool_svc_reconf_ult(void *arg)
 }
 
 static void
-pool_svc_schedule_reconf(struct pool_svc *svc)
+pool_svc_schedule_reconf(struct pool_svc *svc, bool for_chk)
 {
 	struct pool_svc_reconf *reconf = &svc->ps_reconf;
 	enum ds_rsvc_state	state;
 	int			rc;
 
 	D_DEBUG(DB_MD, DF_UUID": begin\n", DP_UUID(svc->ps_uuid));
 
-	if (engine_in_check()) {
+	if (!for_chk && engine_in_check()) {
 		D_DEBUG(DB_MD, DF_UUID": end: skip in check mode\n", DP_UUID(svc->ps_uuid));
```

Comment:
The log message is some misguided for the new patch.

Reply:
@Nasf-Fan, if we see this message, then we are indeed "in check mode" and we will indeed "skip" scheduling a new PS reconf. If we are in check mode and for_chk is true, then we won't print this message. We're good, no? Or, do you want me to add "automatic", such as "end: skip automatic reconf in check mode", to imply that this message won't be printed if this is not an automatic reconf request?

Reply:
I mean that the message "skip in check mode" is inconsistent with "for_chk" parameter, because if "for_chk" is true, the pool_svc_schedule_reconf() is not real skipped. So some suggestion:

1. Adjust the log message, such as you said "skip automatic reconf ...", or
2. Rename "for_chk" as "force", and if "force" is true, then not skip; otherwise if it is in check mode, then skip.

Reply:
I anticipate that we will remove `for_chk` once we replace `engine_in_check` with some real per-pool flag. So I'll only change the log message here. Hopefully that satisfies your requirement.